cli/server.go

```diff
@@ -672,6 +673,54 @@ func server() *cobra.Command {
 	return root
 }
 
+// parseURL parses a string into a URL. It works around some technically correct
+// but undesired behavior of url.Parse by prepending a scheme if one does not
+// exist so that the URL does not get parsed improprely.
+func parseURL(ctx context.Context, u string) (*url.URL, error) {
+	var (
+		hasScheme = strings.HasPrefix(u, "http") || strings.HasPrefix(u, "https")
+	)
```

Comment:
Let's parse for a colon appending these. Otherwise, the weird case of a domain starting with `http` or `https` could conflict here.